lib/services/v2.0/nomination_pools/interfaces.dart

```diff
@@ -174,9 +174,6 @@ mixin KC2NominationPoolsInterface on ChainApiProvider {
     }
   }
 
-  /// @HolyGrease - is there an api to get the min pool amount to create one?
-  /// If not please add one... we need to check amount is above this min whnen creating...
-
```

Comment:
Yes, there is a RPC call for that see `getPoolsConfiguration` which returns `NominationPoolsConfiguration`. 
```dart
class NominationPoolsConfiguration {
  // ...

  /// Minimum bond required to create a pool
  ///
  /// This is the amount that the depositor must put as their initial stake in the pool, as an
  /// indication of "skin in the game"
  ///
  /// This is the value that will always exist in the staking ledger of the pool bonded account
  /// while all other accounts leave
  BigInt minCreateBond;

  // ...
}
```